iocore/net/TLSSessionResumptionSupport.cc

```diff
@@ -160,6 +160,7 @@ TLSSessionResumptionSupport::getSession(SSL *ssl, const unsigned char *id, int l
 #if 0 // This is currently eliminated, since it breaks things in odd ways (see TS-3710)
       ssl_rm_cached_session(SSL_get_SSL_CTX(ssl), session);
 #endif
+      SSL_SESSION_free(session);
```

Comment:
`TLSSessionResumptionSupport::getSession()` looks a function for inbound side for me. How this change is related to the origin session cache?

Reply:
I was checking in this area and found this, guess I should make a separate PR for this.